CSS/04 flex box/04 flex item/03-shrink.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <style>
+    .box {
+      display: flex;
+      flex-direction: column;
+      height: 150px;
+      border: 1px dashed red;
+    }
+
+    .box div {
+      border: 1px solid;
+      flex-basis: auto;
+      width: 160px;
+    }
+
+    .box div {
+      flex-basis: 80px;
+      border: 1px solid;
+    }
```

Comment:
Acho que nesse caso faria mais sentido assim:

```suggestion
    .box div {
      border: 1px solid;
      flex-basis: auto;
      width: 160px;
      flex-basis: 80px;
      border: 1px solid;
    }
```

Reply:
Verdade! Por alguma razão eu fiz duas vezes o mesmo caminho, o que é desnecessário. Obrigado pela correção.